examples/plot_choose_thumbnail.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""
+===========================
+Choosing the thumbnail plot
+===========================
+
+An example to demonstrate how to choose which figure is displayed as the
+thumbnail if the example generates more than one figure. This is done by
+specifying the keyword-value pair ``sphinx_gallery_thumbnail_number = 2`` as a
+comment somewherebelow the docstring in the example file.
+"""
+
+# Code source: Óscar Nájera
+# License: BSD 3 clause
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+def main():
+    x = np.linspace(-1, 2, 100)
+    y = np.exp(x)
+
+    plt.figure()
+    plt.plot(x, y)
+    plt.xlabel('$x$')
+    plt.ylabel('$\exp(x)$')
+
+    # The next line is to pick the figure showing the negative exponential in orange
```

Comment:
Maybe "The next line is to show the second figure in the gallery (plot with negative exponential in orange)"